app/helpers/application_helper.rb

```diff
@@ -43,17 +43,6 @@ def flash_class(level)
 
   ## Devise overrides
 
-  def after_sign_in_path_for(resource)
-    # default to the stored location
-    if resource.is_a?(User) && resource.organization.present?
-      # go to user's dashboard
-      dashboard_path(organization_id: resource.organization.id)
-    else
-      stored_location_for(resource) || new_organization_path
-      # send new users to organization creation page
-    end
-  end
```

Comment:
It was the dead code. It didn't actually override devise's method since it was available in ApplicationController as `helpers.after_sign_in_path_for` and was never invoked. To make it be executed, we would need to either define the method directly in ApplicationController or just include the AppliactionHelper in the ApplicationController.